src/tabs/pid_tuning.html

```diff
@@ -1166,7 +1166,7 @@
                             </tr>
                             <tr class="newFilter dynamicNotch">
                                 <td>
-                                    <input type="number" name="dynamicNotchMinHz" step="1" min="60" max="1000"/>
+                                    <input type="number" name="dynamicNotchMinHz" step="1" min="60" max="250"/>
```

Comment:
If the new max value is "new", it must be changed in the js depending on the version.

Have changed this value between versions?

Reply:
It is changed in the same #9327 PR,
https://github.com/betaflight/betaflight/blob/19fa0f7c23dc9d781e22226d74789274b3a4ca6c/src/main/cli/settings.c#L652

Reply:
I don't know how it can affect old versions, I think 250 has always been a logical limit but it was left at 1000

Reply:
I apologise for changing the upper limit for the minimum.  It was because there seemed no value in allowing a minimum value higher than 250.  So yes, as McGiverGim says, the upper limit for the minimum value needs to be version sensitive in Configurator.  Hopefully it won't change for some time now...